packages/core/test/actions.test.ts

```diff
@@ -1319,6 +1322,39 @@ describe('choose', () => {
 
     expect(service.state.context).toEqual({ answer: 42 });
   });
+
+  it('should be able to use choose actions from within options', () => {
+    interface Ctx {
+      answer?: number;
+    }
+
+    const machine = createMachine<Ctx>(
+      {
+        context: {},
+        initial: 'foo',
+        states: {
+          foo: {
+            entry: 'conditionallyRevealAnswer'
+          }
+        }
+      },
+      {
+        guards: {
+          worstGuard: () => true
+        },
+        actions: {
+          revealAnswer: assign<Ctx>({ answer: 42 }),
+          conditionallyRevealAnswer: choose([
+            { cond: 'worstGuard', actions: 'revealAnswer' }
+          ])
```

Comment:
I have some code that does exactly that but the `revealAnswer` assign action equivalent in my code is never executed. Instead, it just logs `Warning: No implementation found for action type 'xstate.assign'` 🤔